nimbus/src/enrollment.rs

```diff
@@ -473,8 +473,12 @@ impl EnrollmentStatus {
 }
 
 /// Return information about all enrolled experiments.
-pub fn get_enrollments(db: &Database) -> Result<Vec<EnrolledExperiment>> {
-    let enrollments: Vec<ExperimentEnrollment> = db.collect_all(StoreId::Enrollments)?;
+pub fn get_enrollments<'r>(
+    db: &Database,
+    reader: &'r impl Readable<'r>,
```

Comment:
I like that this clearly-read-only function doesn't take a `Writer`, but I don't like that I had to give it an explicit lifetime parameter. My Rust is not strong enough to figure out whether we can avoid it though.